clients/IntermediaryClient.ts

```diff
@@ -175,7 +175,7 @@ export class IntermediaryClient extends StateChannelWallet {
       throw new Error("Transfer amount exceeds owner balance");
     }
 
-    const ownerSig = userOp.signature.slice(0, 65);
+    const ownerSig = userOp.signature.slice(0, 64);
```

Comment:
@lalexgap do this seem right? I was getting an error on the `ethers.concat` line below without this change because it complained about the length of this string

Reply:
As discussed in standup we should probably remove this slice. Due to #109 the signature from the `owner` is already the right length, we don't have to do any slicing.

Reply:
Done in https://github.com/statechannels/SCBridge-Wallet/pull/106/commits/283c08da6cb3ab63b1ee7aabeb0797792d42543a